src/simperium/channel.js

```diff
@@ -293,6 +293,10 @@ export default function Channel( appid, access_token, bucket, store ) {
 		collectionRevisions( channel, id, callback );
 	};
 
+	bucket.hasLocalChanges = function( callback ) {
+		callback( channel.localQueue.hasChanges() );
```

Comment:
Same as in `bucket.js` the first argument of the callback is for an error if there is one.